smokey/src/main/java/frc/robot/control/AutonomousChooser.java

```diff
@@ -158,57 +171,88 @@ else if (this.subsystems.getEveryBotPickerSubsystem() != null) {// cube uses upt
     }
     
     /**
-     * Builds a command list for use in auto routines
-     * @param NodePosition starting position of robot corrosponding to the node. Nodes are numbered from left to right 1- 9 from the drivers perspective
-     * @param Trajectory trajectory to follow out of the community
+     * A method to move the arm to the specified scoring position. 
+     * Caller is responsible for checking that the Arm Subsystem is installed. 
+     * @param height
      * @return command
      */
-    public Command getAutoRoutine (Pose2d NodePosition, Trajectory Trajectory){
+    private Command getArmPositionRoutine (ScoringPosition height){
         SequentialCommandGroup command = new SequentialCommandGroup();
-        command.addCommands(getScoreRoutine(NodePosition));
-        command.addCommands(new DriveTrajectoryCommand(subsystems.getDriveTrainSubsystem(), Trajectory));
+        if (height == ScoringPosition.SCORE_HIGH){
+            command.addCommands(new ArmToLocationCommand(subsystems.getArmSubsystem(), ArmToLocationCommand.ArmLocation.ARM_HIGH_SCORE));
+        }
+        else if (height == ScoringPosition.SCORE_MIDDLE){
+            command.addCommands(new ArmToLocationCommand(subsystems.getArmSubsystem(), ArmToLocationCommand.ArmLocation.ARM_MED_SCORE));
+        }
         return command;
     }
 
     /**
      * Builds a command list for the balance routine, or will not if we toggle it so
      * @param DoBalance the enum for wether the balance routine should be run
-     * @return
+     * @return command
      */
-    public Command getBalanceRoutine (AutonomousBalance DoBalance){
+    private Command getBalanceRoutine (AutonomousBalance DoBalance, Trajectory toRampTrajectory){
         SequentialCommandGroup command = new SequentialCommandGroup();
         if (DoBalance == AutonomousBalance.DO_BALANCE){
-            command.addCommands(new DriveTrajectoryCommand(subsystems.getDriveTrainSubsystem(), trajectories.BehindToOntoRampTrajectory));
+            command.addCommands(new DriveTrajectoryCommand(subsystems.getDriveTrainSubsystem(), toRampTrajectory));
```

Comment:
So wanted to make sure this is intentional, in the event that DoBalance is not AutonomousBalance.DO_BALANCE then the 'toRampTrajectory' will not be completed.  Is this the intent to not do that bit of trajectory?

Reply:
Correct. When we are not balancing, there is a different team balancing and we want to stay out of their way.